version.properties

```diff
@@ -1,3 +1,3 @@
 # Version of produced binaries.
 # The '*' part is automatically resolved by shipkit-auto-version Gradle plugin (https://github.com/shipkit/shipkit-auto-version).
-version=1.3.*
+version=1.4.*
```

Comment:
Is the version number correct? I'm not sure how to stay in sync with Shipkit.

Reply:
Keep it as-is, for now.